Library/Homebrew/utils/curl.rb

```diff
@@ -118,36 +118,40 @@ def curl(*args, print_stdout: true, **options)
       result
     end
 
-    def curl_download(*args, to: nil, partial: true, **options)
+    def parse_headers(headers)
+      return {} unless headers
+
+      headers.split("\n").to_h do |h|
+        partitioned = h.partition(": ")
+        [partitioned.first, partitioned.last]
+      end
+    end
+
+    def curl_download(*args, to: nil, try_partial: true, **options)
```

Comment:
Have all call-sites been updated to use `try_partial` instead of `partial` here?